spring-cloud-stream-samples/rxjava-processor/src/main/java/rxjava/RxJavaTransformer.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package rxjava;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.cloud.stream.annotation.rxjava.EnableRxJavaProcessor;
+import org.springframework.cloud.stream.annotation.rxjava.RxJavaProcessor;
+import org.springframework.context.annotation.Bean;
+
+import rx.Observable;
+
+@EnableRxJavaProcessor
+public class RxJavaTransformer {
+
+	private static Logger logger = LoggerFactory.getLogger(RxJavaTransformer.class);
+
+	@Bean
+	public RxJavaProcessor processor() {
+		return new RxJavaProcessor<String, String>() {
```

Comment:
Generics warning on the `@Bean` method return type.
Or `@SuppressWarning` or specify them. 

From other side this can converted to the Lambda as well:
````java
@Bean
public RxJavaProcessor<String, String> processor() {
     return inputStream -> inputStream.map(...)
                            ...
}
````
And no need to worry about Lambda argument type - just because generics on the return type do the stuff!